AppController/djinn.rb

```diff
@@ -4085,6 +4085,22 @@ def build_uncommitted_changes()
     build_java_appserver if status.include?('AppServer_Java')
   end
 
+  def configure_ejabberd_cert
+    # Replace APPSCALE_CONFIG_DIR/ejabberd.pem with private key and cert
+    # copied from tools. APPSCALE_CONFIG_DIR/ejabberd.pem is not the default
+    # ejabberd.pem file, so we are overwriting changes that we have made.
```

Comment:
Right. The comments seems off. I would remove the last part (the 'so we are ...') or change it to " we don't do nasty things to your lovely manicured system" ...:)